src/routes/album.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const albumController = require("../controllers/album");
+
+const albumRouter = express.Router();
+
+albumRouter.post("/:artists/:id/albums", albumController.createAlbum);
+albumRouter.get("/albums", albumController.readAlbums);
+albumRouter.get("/albums/:id", albumController.getAlbumById);
+albumRouter.put("/albums/:id", albumController.updateAlbum);
+albumRouter.delete("/albums/:id", albumController.deleteAlbum);
```

Comment:
Following from my comment in app.js these routes shouldn't have `"/albums"` in them as that should be added in app.js, i.e.
```
albumRouter.get("/", albumController.readAlbums);
albumRouter.get("/:id", albumController.getAlbumById);
albumRouter.patch("/:id", albumController.patchAlbum);
albumRouter.delete("/:id", albumController.deleteAlbum);
```